### PR TITLE
Add phlex to View Objects & phlex-rails to Rails Presenters

### PR DESCRIPTION
**Pull request by Spone**
Thanks for contributing to the Ruby Toolbox catalog! <3

**Before submitting your pull request:**

- [x] If you're referencing a gem by its GitHub repository (e.g. `rails/rails`), please verify
      that it is _not_ packaged as a Ruby gem. (If it _is_ packaged as a gem, please reference it
      by its gem name instead, e.g. `rails`.)
- [x] Please make sure the projects are listed in case-insensitive alphabetical order
- [x] Make sure the CI build passes, we validate your proposed changes in the test suite :)

### CODE REVIEW

**Comment by Spone** [comment]
:wave: @joeldrapper

File: catalog/Rails_Plugins/rails_presenters.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ projects:
   - komponent
   - meta_presenter
   - oprah
+  - phlex
```
Comment (joeldrapper) [inline catalog/Rails_Plugins/rails_presenters.yml:19]: This should probably be `phlex-rails`, since it's a list of Rails presenters.

Comment (Spone) [reply catalog/Rails_Plugins/rails_presenters.yml:19]:
```suggestion
  - phlex-rails
```

**Comment by stevegeek** [comment]
Should `phlex` also be part of `Code_Organization/View_Objects.yml`?

**Comment by Spone** [comment]
> Should `phlex` also be part of `Code_Organization/View_Objects.yml`?

Yes definitely! I'll update the PR.

**Comment by colszowka** [comment]
Thanks for the contribution @Spone :+1:

**Pull request merged**